### ISSUE TITLE
exporter does not reauthenticate when cf token expires

### ISSUE THREAD

**Issue by frodenas** [cloudfoundry-community/cf_exporter#1]

**Comment by nagelpat** [comment]
Facing the same issue:

```
time="2017-01-09T13:10:44Z" level=error msg="Error while listing organizations: Error requesting organizations Get https://.../v2/organizations: oauth2: cannot fetch token: 401 Unauthorized
Response: {\"error\":\"invalid_token\",\"error_description\":\"Invalid refresh token (expired)
```

**Comment by frodenas** [comment]
@nagelpat I believe this issue is fixed when using version 0.3.0 (now part of the [Prometheus BOSH Release v13.0.0](https://github.com/cloudfoundry-community/prometheus-boshrelease/releases/tag/v13.0.0)). I'm still running some long tests to see if it's really fixed or not, so I'll keep this issue open in the meantime.

**Comment by frodenas** [comment]
I'm going to close this issue as I believe is now fixed. We can reopen if still persists.
